source/Halibut/Transport/SecureWebSocketListener.cs

```diff
@@ -127,7 +127,7 @@ async Task ExecuteRequest(HttpListenerContext listenerContext)
                 var webSocketContext = await listenerContext.AcceptWebSocketAsync("Octopus").ConfigureAwait(false);
                 webSocketStream = new WebSocketStream(webSocketContext.WebSocket);
 
-                var req = await webSocketStream.ReadTextMessage().ConfigureAwait(false); // Initial message
+                var req = await webSocketStream.ReadTextMessage(CancellationToken.None).ConfigureAwait(false); // Initial message
```

Comment:
`ReadTestMessage` used to use a token internal to `WebSocketStream`, which only got cancelled when the whole object was disposed. As this could never happen while executing `ReadTestMessage`, it effectively does not use a CancellationToken.

I updated the signature to allow a CancellationToken to be specified in future usages, but to keep the behaviour the same for this usage, I use `CancellationToken.None`.

Let me know if you think this is actually a behaviour change.

Reply:
We don't want to change the behaviour if Async Halibut is not enabled